### ISSUE TITLE
[Bug] [Android] ContentView in CollectionView does not display anything on Android

### ISSUE THREAD

**Issue by SergejDK** [xamarin/Xamarin.Forms#6461]
### Description
Using a CollectionView where the ItemTemplate has a ContentView with a Label does not display anything in the CollectionView. This seems to work on iOS.

### Steps to Reproduce

1.  Create a CollectionView
2.  Set ItemTemplate:
```csharp
<CollectionView x:Name="colv">
    <CollectionView.ItemTemplate>
         <DataTemplate>
              <ContentView>
                   <Label Text="{Binding Text}"></Label>
              </ContentView>
         </DataTemplate>
    </CollectionView.ItemTemplate>
</CollectionView>
```
3.  Create a class:
```csharp
public class T
{
    public string Text { get; set; }
}
```
4. Create List and set ItemSource of CollectionView
```csharp
List<T> cc = new List<T>();
public MainPage()
{
    InitializeComponent();

    for(int i = 0; i < 10; i++)
    {
        cc.Add(new T { Text = "p" + i });
    }
    colv.ItemsSource = cc;
}
```

### Expected Behavior
Show the items in the collectionview

### Actual Behavior
Nothing is shown in the collectionview

### Basic Information

- Version with issue: 4.0.0.425677 
- Last known good version: -
- IDE: VS Mac, VS
- Platform Target Frameworks: 
  - Android: 9.0  
- Android Support Library Version: 
- Affected Devices: tested with Motorola G6 Plus but seems to not work on any android device.

### Reproduction Link
[https://github.com/SergejDK/XFCollectionViewContentView](https://github.com/SergejDK/XFCollectionViewContentView)

**Comment by LeoJHarris** [comment]
Might be related to: https://github.com/xamarin/Xamarin.Forms/issues/6355

**Comment by hartez** [comment]
This is fixed in 4.1.0.496342-pre2.

**Comment by SergejDK** [comment]
Thanks @hartez ! Works like a charm!
